README.md

```diff
@@ -182,6 +113,50 @@ declare global {
 }
 ```
 
+You will also need to add the following type definitions that are used within the custom `ThemeColors` interface.
+
+```tsx
+export type ThemeOverrides = {
+    avatar: {
+        background: string;
+    };
+
+    bottomNavigation: {
+        inactive: string;
+    };
+    button: {
+        contained: {
+            text: {
+                disabled: string;
+            };
+        };
+    };
+    chip: {
+        outlined: {
+            background: string;
+        };
+        flat: {
+            background: string;
+        };
+    };
+    snackbar: {
+        accent: string;
+    };
+    toggleButtonGroup: {
+        checked: {
+            background: string;
+        };
+    };
+};
+
+export type ThemeOpacity = {
+    disabled: number;
+    divider: number;
+    disabledBackground: number;
+    actionOpacity: number;
+};
```

Comment:
They shouldn't have to define these, they just need to import them from our theme package right? The only thing they should have to define is the global type for the theme.